.devcontainer/devcontainer.json

```diff
@@ -11,9 +11,9 @@
         "terminal.integrated.profiles.linux": {
             "bash": {
                 "path": "/bin/bash",
-                "icon": "terminal-bash",
-            },
-        },
```

Comment:
No, since this is a JSON file, which aren't linted. If you open this file now in our repo, it actually already complains, we just have nothing that can fix it.